package.json

```diff
@@ -71,7 +71,7 @@
     "rollup": "~1.31.0",
     "rollup-plugin-copy": "~3.4.0",
     "rollup-plugin-delete": "~1.2.0",
-    "rollup-plugin-generate-html-template": "~1.7.0",
+    "rollup-plugin-generate-html-template": "1.6.1",
```

Comment:
We need this version of the plugin since the latest version injects into the output index.html all the scripts tags making imposible to lazy load them.

Reply:
We should take this into account when updating rollup in [EX-5336](https://searchbroker.atlassian.net/browse/EX-5336)